pytest_arch/rule.py

```diff
@@ -146,7 +146,11 @@ def check(self, package: str | ModuleType):
         print(f"rule {rule_name}: candidates are {candidates_to_show}")
 
         for c in candidates:
-            imports = all_imports[c].get("direct", []) | all_imports[c].get("transitive", [])
+            imports = (
+                all_imports[c].get("direct", []) | all_imports[c].get("transitive", [])
```

Comment:
I just realised that `[]` needs to be `set()`

Reply:
fixed it #10